pkg/factsengine/entities/fact_value.go

```diff
@@ -192,3 +196,28 @@ func getValue(fact FactValue, values []string) (FactValue, error) {
 		return value, nil
 	}
 }
+
+func Prettify(fact FactValue) (string, error) {
```

Comment:
Should this maybe be a private function?
And it could've been perfectly a struct function as well: 
`func (f *FactValue) Prettify() (string, error)`

Reply:
This was the same way I tried to pursue at the beginning, but it requires a method to be implemented for each implementation of `FactValue`. Instead, this way the function just works :tm: with every concrete type

Reply:
@arbulu89 you cannot add methods to an interface

Reply:
The function is better, you could achieve a semi functional result with a dedicated struct with the method implemented and then embedding with some hassle in every struct you want to prettify

But honestly I don't think it's very ideomatic, seems like some sort of oop abstract superclass with some default methods